Tensile/TensileCreateLibrary.py

```diff
@@ -642,6 +638,38 @@ def filterProcessingErrors(
     return keepKernels, keepSolutions, keepResults
 
 
+def filterBuildErrors(
+    kernels: List[Solution],
+    kernelsWithBuildErrors: Dict[str, int],
+    writerSelectionFn: Callable[[str], KernelWriterSource | KernelWriterAssembly],
```

Comment:
Use `Optional`

Reply:
Adding `Union` instead of `Optional` which is equivalent to `Union[YourType, None]`